lib/micro/case/result.rb

```diff
@@ -25,6 +25,7 @@ def self.transitions_enabled?
       def initialize(transitions_mapper = Transitions::MapEverything)
         @__accumulated_data = {}
         @__accessible_attributes = {}
+        @__is_unknown = true
```

Comment:
@tomascco is a code style of the project, but, I like to order the lines by size to improve the code reading/scanning (by human eyes).

```suggestion
        @__is_unknown = true
        @__accumulated_data = {}
        @__accessible_attributes = {}
```

Reply:
I would never think that this was intentional, that's pretty nice.